docs/source/guides/using_dask_entitysets.ipynb

```diff
@@ -194,18 +194,37 @@
     "\n",
-    "By default, when Woodwork checks that pandas DataFrames have unique index values. Because performing this same check with Dask would require an expensive compute operation, this check is not performed when adding a Dask DataFrame to an `EntitySet`. When using Dask DataFrames, users must ensure that the supplied index values are unique.\n",
+    "By default, Woodwork checks that pandas DataFrames have unique index values. Because performing this same check with Dask would require an expensive compute operation, this check is not performed when adding a Dask DataFrame to an `EntitySet`. When using Dask DataFrames, users must ensure that the supplied index values are unique.\n",
     "\n",
     "When using a pandas DataFrames, the ordering of the underlying DataFrame rows is maintained by Featuretools. For a Dask DataFrame, the ordering of the DataFrame rows is not guaranteed, and Featuretools does not attempt to maintain row order. If ordering is important, close attention must be paid to any output to avoid issues.\n",
     "\n",
     "### EntitySet Limitations\n",
     "\n",
     "When creating a Featuretools `EntitySet` that will be made of Dask DataFrames, all of the DataFrames used to create the `EntitySet` must be of the same type, either all Dask DataFrames or all pandas DataFrames. Featuretools does not support creating an `EntitySet` containing a mix of Dask and pandas DataFrames.\n",
     "\n",
-    "Additionally, the `EntitySet.add_interesting_values()` method is not supported when using a Dask `EntitySet`. If needed, users can manually set `interesting_values` for DataFrames by assigning them directly with syntax similar to this: `es[\"dataframe_name\"].ww[\"variable_name\"].ww.metadata[\"interesting_values\"] = [\"Value 1\", \"Value 2\"]`.\n",
+    "Additionally, ``EntitySet.add_interesting_values()`` cannot be used in Dask EntitySets to find interesting values; however, it can be used set a column's interesting values with the `values` parameter."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a3c6b9b8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "values_dict = {'values': [12, 103]}\n",
+    "es.add_interesting_values(dataframe_name='dask_input_df', values=values_dict )\n",
```

Comment:
Super minor nitpick, but there is an extra space between `values_dict` and the closing `)`.